src/shared/components/structureViewer/StructureViewerPanel.tsx

```diff
@@ -508,6 +530,24 @@ export default class StructureViewerPanel extends React.Component<IStructureView
         }
     }
 
+    @computed get structureViewerBounds(): {width: number|string, height: number|string} {
+        let width: number|string;
+        let height: number|string;
+
+        // if 3Dmol container div is not initialized yet, just set to a default value: width=auto; height=350
+        // otherwise toggle the size
+        if (this.isIncreasedSize) {
```

Comment:
`@computed` does not work as desired if the condition is written this way:
```JavaScript
if (this._3dMolDiv) {
	width = this.isDoubleSize ? this._3dMolDiv.offsetWidth * 2 : this._3dMolDiv.offsetWidth / 2;
	height = ...
} else {
	width = "auto";
	height = 350;
}
```
This is probably because `_3dMolDiv` is not observable.